web/src/top_left_corner.js

```diff
@@ -10,6 +10,16 @@ export function update_starred_count(count) {
     ui_util.update_unread_count_in_dom($starred_li, count);
 }
 
+export function update_scheduled_count(count) {
+    const $scheduled_li = $(".top_left_scheduled_messages");
+    if (count > 0) {
+        $scheduled_li.removeClass("hidden-on-zero");
+    } else {
+        $scheduled_li.addClass("hidden-on-zero");
+    }
+    ui_util.update_unread_count_in_dom($scheduled_li, count);
+}
```

Comment:
Hmm, why the name `hidden-on-zero`? That name to me reads as "We've written some fancy CSS that will auto-hide this when the value is zero". 

I guess `$scheduled_li.hide()` and `$scheduled_li.show()` would normally be the right answer; I think that'll naturally have the right outcome for spectators, because they'll always have 0 scheduled messages anyway; might deserve a comment on that detail.a